core/src/bytecode/ast/pattern/mod.rs

```diff
@@ -0,0 +1,234 @@
+//! Pattern matching and destructuring of Nickel values.
+use std::collections::{hash_map::Entry, HashMap};
+
+use super::{Annotation, Ast, Number};
+
+use crate::{identifier::LocIdent, parser::error::ParseError, position::TermPos};
+
+pub mod bindings;
+
+/// A small helper to generate a
```

Comment:
to generate a?

Reply:
Seems to be an old leftover, which was already orphan in `term::pattern`. Got rid of it in both places.